### PR TITLE
Fixes #29695 - Add svg file support

### PR DESCRIPTION
**Pull request by fhbash**
Adding svg file support to webpack.config file, so we can use svg image
files on react components.

Signed-off-by: Fellipe Henrique <fpedrosa@redhat.com>

### CODE REVIEW

File: config/webpack.config.js [hunk 1]
```diff
@@ -145,7 +145,7 @@ module.exports = env => {
           })
         },
         {
-          test: /(\.png|\.gif)$/,
+          test: /(\.png|\.gif|.svg)$/,
```
Comment (adamruzicka) [inline config/webpack.config.js]: Shouldn't there be a leading backward slash in front of the dot?

Comment (fhbash) [reply config/webpack.config.js]: You're right... fixed!

File: config/webpack.config.js [hunk 2]
```diff
@@ -145,7 +145,7 @@ module.exports = env => {
           })
         },
         {
-          test: /(\.png|\.gif)$/,
+          test: /(\.png|\.gif|\.svg)$/,
```
Comment (ekohl) [inline config/webpack.config.js]: Perhaps it should be shorted:
```suggestion
          test: /\.(png|gif|svg)$/,
```

**Comment by xprazak2** [comment]
@fellipeh, I have created an issue in [redmine](https://projects.theforeman.org/issues/29695) to track this. Could you change your commit message to 'Fixes #29695 - Add svg file support' please? That should make the check for redmine issues pass...

**Comment by ekohl** [comment]
ok to test

**Review by xprazak2 — APPROVED** [review]

**Comment by xprazak2** [comment]
LGTM. @tbrisker I think this is a low-risk, could you please ack this if you agree?

**Review by tbrisker — APPROVED** [review]
Thanks @fellipeh !

**Pull request merged**